python/spacewalk/satellite_tools/spacewalk-data-fsck

```diff
@@ -357,6 +359,27 @@ def is_srpm(file):
     else:
         return False
 
+def is_orphaned_rpm(package_id: int) -> bool:
+    if not isinstance(package_id, int):
+        # Prevent possible SQL injections
+        raise ValueError("package_id must be an integer")
+
+    query = f"""
+    SELECT EXISTS (
+      SELECT 1 FROM rhnPackage p
+      JOIN rhnChannelPackage c ON c.package_id = p.id WHERE p.id = {package_id}
```

Comment:
Better to use `:package_id` and provide in `execute(package_id=package_id=`
In case quoting is required, it would do it for you.